src/sql/workbench/browser/modelComponents/declarativeTable.component.ts

```diff
@@ -258,49 +257,51 @@ export default class DeclarativeTableComponent extends ContainerBase<any, azdata
 
 	private static ACCEPTABLE_VALUES = new Set<string>(['number', 'string', 'boolean']);
 	public setProperties(properties: azdata.DeclarativeTableProperties): void {
-		const basicData: any[][] = properties.data ?? [];
-		const complexData: azdata.DeclarativeTableCellValue[][] = properties.dataValues ?? [];
-		let finalData: azdata.DeclarativeTableCellValue[][];
-
-		finalData = basicData.map(row => {
-			return row.map((value): azdata.DeclarativeTableCellValue => {
-				if (DeclarativeTableComponent.ACCEPTABLE_VALUES.has(typeof (value))) {
-					return {
-						value: value
-					};
-				} else {
-					return {
-						value: JSON.stringify(value)
-					};
-				}
+		if ('data' in properties || 'dataValues' in properties) {
```

Comment:
Why are you adding this?

Reply:
I think I found the root cause that made me adding this.